index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <title>@Gopxfs Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+<body>
+  <div class ="page-flex">
+    <header class="header-flex">
+        <div class="header-logo-placeholder">
+            <p class="header-logo">John Doe</p>
+        </div>
+        <button type="button" class="header-button">
+            <span class="header-menu-placeholder">
+                <span class="header-menu-icon"></span>
+                <span class="header-menu-icon"></span>
+                <span class="header-menu-icon"></span>
+            </span>
+        </button>
+    </header>
+
+    <div class="headline-flex">
+        <div class="headline-heading-placeholder">
+            <h1 class="headline-heading-name">Laura Jong</h1>
+            <h2 class="headline-heading-occupation">Software Developer</h2>
+        </div>
+        <div class="headline-description-placeholder">
+            <p class="headline-description">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        </div>
+    </div>
+  </div>
```

Comment:
- [ ] Headline is looking good! However, it's better to wrap it with a `section` tag for the semantics of your project.